src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1897,6 +1988,100 @@ private long EnforceConfigurationFor(DocumentsOperationContext context, string i
             }
         }
 
+        private bool AdoptOrphanedFor(DocumentsOperationContext context, string id)
+        {
+            using (DocumentIdWorker.GetSliceFromId(context, id, out var lowerId))
+            using (GetKeyPrefix(context, lowerId, out var lowerIdPrefix))
+            {
+                var collectionName = GetCollectionFor(context, lowerIdPrefix);
+                if (collectionName == null)
+                {
+                    if (_logger.IsInfoEnabled)
+                        _logger.Info($"Tried to delete revisions for '{id}' but no revisions found.");
+                    return false;
+                }
+
+
+                if (ShouldCreateDeleteRevision(context, lowerId, lowerIdPrefix, collectionName, out var table, out var lastRevision))
+                {
+                    var lastModifiedTicks = _database.Time.GetUtcNow().Ticks;
+                    var newEtag = _database.DocumentsStorage.GenerateNextEtag();
+                    var cv = _documentsStorage.GetNewChangeVector(context, newEtag);
+                    CreateDeleteRevision(context, table, id, collectionName, cv, lastModifiedTicks, lastRevision.Flags);
+                    return true;
+                }
+
+                return false;
+            }
+        }
+
+        internal void DeleteLastRevisionFor(DocumentsOperationContext context, string id)
+        {
+            var collectionName = new CollectionName("Users");
```

Comment:
'Users' ?!

Reply:
if this is only for tests/debug, please make a partial class `RevisionsStorage.Debug` and move it there

Reply:
Yes It is onlly for tests..
I preferred doing that instead of turning private fields/methods into 'internal'

Reply:
Done